signac_dashboard/modules/document_editor.py

```diff
@@ -28,7 +28,13 @@ def __init__(
         template="cards/document_editor.html",
         **kwargs,
     ):
-        super().__init__(name=name, context=context, template=template, **kwargs)
+        self._enabled_contexts = {"JobContext"}
```

Comment:
I think you want this to be defined at the class level, and I think you want it to be named `_supported_contexts`. "Enabled" already has a different meaning (whether it's turned on or off) and this is really a matter of "support." Also, this can't change per instance of the class, which indicates it should be class-level.

```python
class DocumentEditor(Module):
    _supported_contexts = {"JobContext"}

    def __init__(self, ...):
        pass
```

If you define this for the class, then you can also check it on the class itself, rather than just on instances of the class.

```python
>>> class A:
...   _class_var = "asdf"
... 
>>> A._class_var
'asdf'
>>> A()._class_var
'asdf'
```